src/bg_daemon/main.m

```diff
@@ -0,0 +1,50 @@
+#import <Foundation/Foundation.h>
+#import <AppKit/AppKit.h>
+
+int main(int argc, const char * argv[]) {
+  int ret = 0;
+
+  @autoreleasepool {
```

Comment:
I'm not sure why this is showing like this here, it looks fine in my editor:

![screen shot 2015-04-03 at 9 18 23 pm](https://cloud.githubusercontent.com/assets/375307/6991409/fcd3af00-da46-11e4-825e-6cd9f186b40f.png)